src/main/java/se/iths/SpringbootGroupProject/SpringbootGroupProjectApplication.java

```diff
@@ -2,10 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
 @SpringBootApplication
 @EnableJpaAuditing
+@EnableCaching
```

Comment:
Probably won't affect the application, but I don't think you need @EnableCaching here since you have done it on CachingConfig which is annotated with @Configuration. 
Will approve the PR either way :)

Reply:
I was also uncertain about this. Should I remove it? 🤔